src/__tests__/Reducers.test.js

```diff
@@ -0,0 +1,64 @@
+import reducer from '../reducers';
+import * as actions from '../actions';
+import {
+  RECEIVE_LOCATION,
+  CHANGE_STATUS,
+  INPUT_CHANGE,
+} from '../constants';
+
+describe('reducers', () => {
+  it('returns the initial state', () => {
+    expect(
+      reducer(undefined, {})).toEqual({
+        locations: [],
+        message: [],
+      },
+    );
+  });
+
+  it('handles RECEIVE_LOCATION', () => {
+    const date = new Date();
+    const name = 'Richmond, VA';
+    const aqi = 62;
+    const description = 'Fair air quality';
+    const color = 'rgb(181, 226, 30)';
+    const expectedAction = {
```

Comment:
the action isn't part of your expectation, the state of your store after you call that action on your reducer is your expectation.